frontend/src/App.vue

```diff
@@ -72,30 +72,31 @@
           <div class="content has-text-right has-text-centered-mobile is-size-2-mobile">
             <p>
               <a href="https://www.sysbio.se" title="SysBio">
-                <img src="./assets/sysbio-logo.png" />
+                <img class="my-0 mx-2" src="./assets/sysbio-logo.png" />
               </a>
               <a href="http://www.chalmers.se" title="Chalmers University of Technology">
-                <img src="./assets/chalmers.png" />
+                <img class="my-0 mx-2" src="./assets/chalmers.png" />
               </a>
               <a href="https://kaw.wallenberg.org/" title="Knut and Alice Wallenberg Foundation">
-                <img src="./assets/wallenberg.gif" />
+                <img class="my-0 mx-2" src="./assets/wallenberg.gif" />
               </a>
               <a href="https://www.kth.se/en/bio/centres/wcpr" title="CBH | KTH Royal Institute of Technology">
-                <img src="./assets/wpcr.jpg" />
+                <img class="my-0 mx-2" src="./assets/wpcr.jpg" />
               </a>
               <a href="https://nbis.se/">
-                <img src="./assets/nbislogo-green.png" title="National Bioinformatics Infrastructure Sweden" />
+                <img class="my-0 mx-2" src="./assets/nbislogo-green.png"
+                  title="National Bioinformatics Infrastructure Sweden" />
```

Comment:
I think the indentation is off here. Not a big deal but would generate a linter warning.